src/engine/strat_engine/tests/real.rs

```diff
@@ -181,7 +181,7 @@ fn make_linear_test_dev(devnode: &Path, start: Sectors, length: Sectors) -> Line
         get_dm(),
         DmName::new(&format!("stratis_test_{}", Uuid::new_v4())).expect("valid format"),
         None,
-        table.clone(),
```

Comment:
I think this is a stray edit, although it must be correct. Can you take it out of this PR and push it to a PR for November?

Reply:
I added this due to accidentally using nightly for clippy and finding out we'll get an error soon. Do you still want me to separate this out?

Reply:
Go clippy! Yes please, toss it into November.

Reply:
Reverted.